python/cudf/cudf/tests/test_scalar.py

```diff
@@ -387,34 +381,6 @@ def test_scalar_invalid_implicit_conversion(cls, dtype):
             cls(slr)
 
 
-@pytest.mark.parametrize("value", SCALAR_VALUES + DECIMAL_VALUES)
-@pytest.mark.parametrize(
-    "decimal_type",
-    [cudf.Decimal32Dtype, cudf.Decimal64Dtype, cudf.Decimal128Dtype],
-)
-def test_device_scalar_direct_construction(value, decimal_type):
```

Comment:
I guess you'll add something like this test back in #17054. Should we add a TODO comment here or a note in the issue?

Reply:
Yeah a test like this will be added back in https://github.com/rapidsai/cudf/issues/17054